script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -1383,27 +1412,28 @@ A suspicious pair like us are bound to be remembered.
 [sha:00579241620b037aa6b9648ad830dfa23fbcf897]{
 -- Page 899, Offset 8053.
 -- ホテルからできるだけ離れ、総耶駅行きのバスに乗りこむ。
-I board a bus headed toward Souya Station to get as far away from the hotel as possible.
+Trying to put as much distance as possible between us and the hotel,
+ I board a bus headed for Souya station.
 }
 [sha:2188d668e61f4d7f874f8af1b352e00ad8876d1a]{
 -- Page 899, Offset 8054.
 -- 早朝という事が幸いして、人目に付く事はなかった。
-Fortunately, it's early in the morning so we didn't attract any attention.
+Fortunately, it's early in the morning so we don't attract any attention.
 }
 [sha:ddd01cc884b2d8a9877411abe17407ede9c872c8]{
 -- Page 900, Offset 8055.
 -- 裏の非常口は一昨日のままだった。
-The emergency exit at the back of the building was still the same as it was the day before yesterday.
+The emergency exit at the back of the building is still the same as it was the day before yesterday.
 }
 [sha:d56e4d1bd5a4de2571c18938741a9afa4ff52e43]{
 -- Page 900, Offset 8056.
 -- 住人も管理人もここの鍵が“切れている”事に気づいていないのだろう。
-The residents and managers must not have noticed that the lock had been 'cut'.
+The residents and managers must not have noticed that the lock has been 'cut'.
```

Comment:
I still think had is correct, both the actions of 'not noticing' and 'cutting the lock' happened in the past.

Reply:
I feel like `has` here better captures "it was cut _and remains cut_", whereas when I read "the lock had been cut" it feels more like this is an incident report / it has been discovered and remedied